VariablesHomework/VariablesHomework.cs

```diff
@@ -8,50 +8,67 @@ public class Variables
         [TestMethod]
         public void Declaration()
         {
+            int number = 25;
             Assert.IsTrue(number == 25); // Make this pass
+
+            bool boolean = true;
             Assert.IsTrue(boolean == true); // Make this pass
+
+            double decimalValue = 99.9;
             Assert.IsTrue(decimalValue == 99.9); // Make this pass
+
+            float floatingDecimalValue = 25.4f;
             Assert.IsTrue(floatingDecimalValue == 25.4f); // Make this pass
+
+            string words = "I am a programmer now";
             Assert.IsTrue(words == "I am a programmer now"); // Make this pass
+
+            char character = 'z';
             Assert.IsTrue(character == 'z'); // Make this pass
         }
 
         [TestMethod]
         public void Assignment()
         {
-            int i = 0;
-            Assert.IsTrue(i == 25); // Make this pass
+            int t = 25;
+            Assert.IsTrue(t == 25); // Make this pass
 
-            Assert.IsTrue(i == 100); // Then make this pass
+            int y = 100;
```

Comment:
Don't create a new variable here. We want to assign a new value into the same variable `i`.
The same with `x` (reuse `d` instead), `p` (reuse `s`), and `g` (reuse `c`)

Reply:
How do I allow it to be assigned the same variable if it was previously assigned without running into an error?

Thanks for your feedback.

Reply:
The same way that `f` gets reassigned (and further down in the Operators block, the way `z` gets reassigned.

```C#
double myVariable;  // Creates myVariable, but doesn't give it a value.
double myOtherVariable = 4.2;  // Creates myOtherVariable and initializes it to the value 4.2

myVariable = 2.7;  // Not creating myVariable, so don't name the type. Just assign a value to it.
myOtherVariable = 7.2;  // Assign a new value to myOtherVariable
```

Reply:
Ok, thanks for that explanation.